go/vt/topo/etcd2topo/server.go

```diff
@@ -34,13 +34,21 @@ We follow these conventions within this package:
 package etcd2topo
 
 import (
+	"flag"
 	"strings"
 	"time"
 
 	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/pkg/transport"
 	"vitess.io/vitess/go/vt/topo"
 )
 
+var (
+	clientCertPath = flag.String("topo_etcd2_tls_cert", "", "the client cert to use to connect to the etcd topo server, requires topo_etcd2_tls_key, enables TLS")
```

Comment:
Sorry, I have to reverse my earlier comment. It turns out the precedent is that we prefix our flags with `topo_etcd` and `topo_zk` even though these are for the etcd2 and zk2 topo implementations.

Reply:
Updated.